pages_builder/generate_pages.py

```diff
@@ -161,8 +161,15 @@ def render_page(self, root, file):
                 for index, example in enumerate(partial["examples"]):
                     grid = partial.get('grid')
                     if isinstance(example, dict):
+                        # if the example has some html it needs to be displayed, cache it and remove from the parameters example
+                        surrounding_html = example.get('surrounding_html', None)
+                        if surrounding_html:
+                            del example['surrounding_html']
```

Comment:
Bit late now, but you could do a `.pop('surrounding_html', None)` and not have to do a bunch of this.